privacyidea/api/token.py

```diff
@@ -161,6 +163,8 @@ def init():
     :jsonparam type: the type of the token
     :jsonparam tokenrealm: additional realms, the token should be put into
     :jsonparam otplen: length of the OTP value
+    :jsonparam registrationcode_length: the length of the generated registration code
```

Comment:
Usually we do not have the token specific parameters in the API descritpion.
Please note my comment on changing the name of the parameters.

Reply:
Removed.